testfiles/badger2Spec

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Unless I'm mistaken it doesn't look like you've used this test file anywhere.

Reply:
Note: If you were to actually run tests end-to-end tests using badger2 like this it would fail because there is not yet a badger2 plugin for go-ipfs. Once you have a draft PR up for that you can link it into this one and tests will be able to pass

Reply:
I don't see offhand any natural place to put a badger2 test. I think we should add a set of tests that just tries converting from all primitive datastores (flatfs, leveldb, memory, badger, badger2) to in memory leveldb and then back again.